examples/federated-learning/tff/nvflare/base-client/base-client.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    checkov.io/skip1: CKV_K8S_21=Default namespace used
+    checkov.io/skip2: CKV_K8S_43=Image version used instead of digest
+    checkov.io/skip5: CKV_K8S_8=Liveness probes not set
+    checkov.io/skip9: CKV_K8S_40=No high UID
+    checkov.io/skip10: CKV_K8S_22=No read-only filesystem
+    checkov.io/skip12: CKV_K8S_38=No service account token
+    checkov.io/skip14: CKV_K8S_9=Readiness probes not set
+    checkov.io/skip16: CKV_K8S_31=No seccomp profile
+  name: nvflare-client
+  labels:
+    run: nvflare-client
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: nvflare-client
+  template:
+    metadata:
+      labels:
+        run: nvflare-client
+      annotations:
+        gke-gcsfuse/volumes: "true"
+    spec:
+      containers:
+        - name: nvflare
+          image: europe-west1-docker.pkg.dev/lgu-demos/nvflare/nvflare-tensorflow:1.0.0
```

Comment:
Check this because it references an image that doesn't exist outside your environment.